lib/sqlitex/row.ex

```diff
@@ -22,10 +22,21 @@ defmodule Sqlitex.Row do
     nil
   end
 
+  # date is empty ""
+  defp translate_value({"", "date"}), do: nil
+
   defp translate_value({date, "date"}) when is_binary(date), do: to_date(date)
 
+  # time is empty ""
+  defp translate_value({"", "time"}), do: nil
+
   defp translate_value({time, "time"}) when is_binary(time), do: to_time(time)
 
+  # datetime is empty ""
+  defp translate_value({"", "datetime"}) do
+    {nil, nil}
```

Comment:
I think this should be nil, rather than {nil, nil} to mirror the behaviour when the column is NULL:

```
iex(6)> Sqlitex.with_db("temp_test.db", fn (db) ->
...(6)>   Sqlitex.exec(db, "CREATE TABLE x (dt DATETIME);")
...(6)>   Sqlitex.exec(db, "INSERT INTO x (dt) VALUES (NULL);")
...(6)>   Sqlitex.query(db, "SELECT * FROM x;")
...(6)> end)
[[dt: nil]]
```

Also means that we can insert the value we get out back into the database without any effort: sqlitex knows how to insert `nil`, but not `{nil, nil}`